README.md

```diff
@@ -60,6 +60,10 @@ cd <path/to/SMARTS>
 # Follow the instructions given by prompt for setting up the SUMO_HOME environment variable
 bash utils/setup/install_deps.sh
 
+# Alternatively you can use the -y option to allow the default accept of installation to prevent timeout from waiting for user input. 
+# Though you still need to follow the SUMO_HOME environment variable instructions after the script finishes executing.
+bash utils/setup/install_deps.sh -y
+
```

Comment:
I think we can condense the instructions, this might be enough: 
```suggestion
# Install the system requirements then follow the instructions given by prompt for setting up the SUMO_HOME environment variable
# You can use the -y option to allow the default accept of installation to prevent timeout from waiting for user input. 
bash utils/setup/install_deps.sh
```